app/editor/src/features/content/form/interfaces/IContentForm.ts

```diff
@@ -1,62 +1,30 @@
-import {
-  ContentStatusName,
-  ContentTypeName,
-  IContentActionModel,
-  IContentLabelModel,
-  IContentLinkModel,
-  IContentTagModel,
-  IContentTonePoolModel,
-  IContentTopicModel,
-  IFileReferenceModel,
-  IMediaTypeModel,
-  ITimeTrackingModel,
-  IUserModel,
-  IWorkOrderModel,
-} from 'tno-core';
+import { IContentModel, IWorkOrderModel } from 'tno-core';
 
-export interface IContentForm {
-  id: number;
+export interface IContentForm
```

Comment:
This is a better way to create a new interface that is nearly identical to another one.